### ISSUE TITLE
Linking datasets

### ISSUE THREAD

**Issue by djwesto** [folklabs/retail_points_organizations#1]
I see you've assigned a Wikidata URL to the rows which can be referenced on Wikidata.

What do you suggest we do about those without a Wikidata URL? One nice thing would be to add them.

Another would be to flag them to the organisations.

I favour the former.

**Comment by pmackay** [comment]
Yes, I'll add them at some point, fortunately only a small number missing.
